doc/source/data/api/input_output.rst

```diff
@@ -63,6 +63,9 @@ TFRecords
 
 .. autofunction:: ray.data.read_tfrecords
 
+.. autofunction:: ray.data.Dataset.write_tfrecords
+    :noindex:
+
```

Comment:
We should also add `write_tfrecords` to the write-side of our datasource compatibility matrix: https://docs.ray.io/en/master/data/dataset.html#supported-output-formats

https://github.com/ray-project/ray/blob/master/doc/source/data/dataset.rst#supported-output-formats

Reply:
Maybe worthing adding to the "Saving Datasets" section of the "Consuming Datasets" feature guide: https://docs.ray.io/en/master/data/consuming-datasets.html#saving-datasets